package.json

```diff
@@ -92,6 +92,7 @@
         "vue-deepset": "^0.6.3",
         "vue-events": "^3.1.0",
         "vue-monaco": "^1.2.1",
+        "vue-pagination-2": "^3.1.0",
```

Comment:
@pmoskrojas is this being used anywhere? I don't see `import Pagination from 'vue-pagination-2';`

Reply:
This is necessary for ./resources/js/components/common/Pagination.vue and other files which use Bootstrap too like ./resources/js/components/common/mixins/datatable.js. Generally for improve performance and current behavior.